build/Shared/SdkAnalysisLevelMinimums.cs

```diff
@@ -0,0 +1,10 @@
+/// <summary>
+/// Defines constants that specify the minimum SDK analysis level required to enable new features.
+/// </summary>
+namespace NuGet.Shared
+{
+    internal static class SdkAnalysisLevelMinimums
+    {
+        public const string HttpErrorSdkAnalysisLevelMinimumValue = "9.0.100";
```

Comment:
I know it's not ready yet, but you can probably use a NuGetVersion type that way we're not reparsing it every time.